README.md

```diff
@@ -1,6 +1,6 @@
 # metatests
 
-[![TravisCI](https://travis-ci.org/metarhia/metatests.svg?branch=master)](https://travis-ci.org/metarhia/metatests)
+![CI Status Badge](https://github.com/metarhia/metatests/workflows/Testing%20CI/badge.svg?branch=master)
```

Comment:
```suggestion
[![CI Status Badge](https://github.com/metarhia/metatests/workflows/Testing%20CI/badge.svg)](https://github.com/metarhia/metatests/actions)
```

Reply:
It was just consistent with `@metarhia/common`, we should probably change it there too.

I think the link should be `https://github.com/metarhia/metadoc/actions?query=workflow%3A%22Testing+CI%22+branch%3Amaster` though?

Reply:
Yes

Reply:
`https://github.com/metarhia/metatests/actions?query=workflow%3A%22Testing+CI%22+branch%3Amaster`